tests/test_ci_minimal.py

```
@@ -0,0 +1,100 @@
+sys.path.insert(0, str(Path(__file__).parent.parent))
+
+# Detect CI environment
+IS_CI = os.environ.get("CI", "false").lower() == "true" or os.environ.get("GITHUB_ACTIONS", "false").lower() == "true"
+
+
+def test_basic_imports():
+    """Test that basic Python libraries work."""
+    import json
+    import re
+    from pathlib import Path
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)